source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/op_exx_lcao.hpp

```diff
@@ -0,0 +1,119 @@
+#ifndef OPEXXLCAO_HPP
+#define OPEXXLCAO_HPP
+
+#ifdef __EXX
+
+#include "op_exx_lcao.h"
+#include "module_ri/RI_2D_Comm.h"
+#include "module_hamilt_pw/hamilt_pwdft/global.h"

```

Comment:
I suggest not to include `global.h` in .h or .hpp files.
It will include many global classes.
I suggest that you can move Global classes definition of EXX to another file.

Reply:
`GlobalC::restart` and `GlobalC::exx_info` are in `global.h`